apps/i18n/common/en_us.json

```diff
@@ -1012,6 +1012,7 @@
   "sharingAgePrompt": "Please select an age.",
   "sharingBlockedByTeacher": "Sorry, you do not have permissions to share this project. If you want to be able to share your project, please ask your teacher to enable sharing of App Lab / Game Lab / Web Lab projects for your section from the 'Manage students' tab in their dashboard. They can do this by adding the project sharing column from the Actions settings menu.",
   "show": "Show",
+  "showStudentName": "Student name",
```

Comment:
What's the reason for the "show" part? We would typically just say `studentName` if this text is just a label and not an action.

Reply:
Fixed.  Changed showStudentName to studentNameHeader
Thanks.